src/main/scala/org/globalforestwatch/features/ValidatedFeatureRDD.scala

```diff
@@ -22,18 +23,21 @@ object ValidatedFeatureRDD {
    * Reads features from source and optionally splits them by 1x1 degree grid.
    * - If the feature WKB is invalid, the feature will be dropped
    * - If there is a problem with intersection logic, the erroring feature id will propagate to output
+    *  If gfwProAddCentroid is true, then add the centroid of GFWPro location to the
+    *  featureId if its locationId is not -1.  This is used for GFWProDashboard.
    */
   def apply(
     input: NonEmptyList[String],
     featureType: String,
     filters: FeatureFilter,
-    splitFeatures: Boolean
+    splitFeatures: Boolean,
+    gfwProAddCentroid: Boolean = false
```

Comment:
Very minor, but is there a reason to include `gfwPro` in the name vs. just call it `addCentroid`?

Reply:
For efficiency, I am only computing the centroid of the whole geometry if the locationId of the GFWProFeatureId is not -1.  [We don't need the centroid if it's a large dissolved list with locId of -1, since for the dissolved geometries, we do want all the different intersecting GADMs.]  So, this option is GFWPro-specific, since I have to deconstruct the GFW FeatureId to check the location Id.